build/packaging/suseconnect-ng.changes

```diff
@@ -4,6 +4,8 @@ Thu Apr 25 15:39:00 UTC 2024 - Felix Schnizlein <fschnizlein@suse.com>
 - 1.9.0 (unreleased)
   * Build zypper-migration and zypper-packages-search as standalone
     binaries rather then one single binary
+  * Include /etc/products.d in directories whose content are backed
+    up and restored if a zypper-migration rollback happens.
```

Comment:
```suggestion
    up and restored if a zypper-migration rollback happens. (bsc#1219004)
```
So maintenance can auto detect the fix for the bug :)

Reply:
Oops yes, will do ;-)